### PR TITLE
Change Active Record dependency to Active Model

### PR DESCRIPTION
**Pull request by SauloSilva**
ref. #22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,23 +27,21 @@ source 'http://rubygems.org'
...
 gem 'sinatra-contrib', '1.4.6'
-gem 'sinatra-active-record', '2.0.4'
 gem 'sinatra-active-model-serializers', '0.2.0'
 ```
 
 ## Configure
 
-First you have to being with the active record configured in your system environment correctly. If the environment is not set up, see their [documentation](https://github.com/janko-m/sinatra-activerecord#sinatra-activerecord-extension).
-After is simply do require from our library on your `application.rb`, eg.
+It's very easy. Look by example:
```
Comment (adrianotadao) [inline README.md]: @SauloSilva Maybe the correct way do say that is: `Look at the example`.

File: README.md [hunk 2]
```diff
@@ -62,13 +60,13 @@ set :active_model_serializers, { root: false }
 
 #### serializers_path
 
-By default this attribute is set up to look for the serializers from your project in "* app / serializers *". Whether you have a different environment you can set up by inserting the path of the string, eg.
+By default path is `app/serializers`. If you need a different path you can set your path, eg.
```
Comment (adrianotadao) [inline README.md]: @SauloSilva 

`By default, path is`

it will be? hahhaha

**Comment by adrianotadao** [comment]
I've not tested in my computer but, I can't see anything wrong... good job!

File: README.md (hunk 2 above)
Comment (SauloSilva) [inline README.md]: I agree :+1:

File: README.md (hunk 1 above)
Comment (SauloSilva) [inline README.md]: It's better, thanks

**Comment by SauloSilva** [comment]
Thanks man, if you agree, you could merge this PR and congrats for review (=

**Comment by SauloSilva** [comment]
@adrianotadao :heart: :heart: :heart:

**Comment by adrianotadao** [comment]
@SauloSilva sorry man, I have a girlfriend hahahahahaha

**Comment by SauloSilva** [comment]
@adrianotadao it's a pity hehehe

**Pull request merged**